src/styles/blocks/catalog.scss

```diff
@@ -21,5 +21,14 @@
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
+
+    &:first-child {
+      transition: transform 300ms ease, color 300ms ease;
+
+      &:hover {
+        transform: scale(1.2);
+        color: #34568b;
```

Comment:
Please, look through the task, as I understood these effects have to be added to all cards, not only to the first one.

Reply:
Ohh, yes. My bad, thanks 😉